CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `MMD.compute` no longer returns `nan`. (https://github.com/gchq/coreax/issues/855)
+- CaratheodoryRecombination now generates correct coresets on GPU machines
```

Comment:
Should this now be updated to mention the fix for all the recombination solvers, and should there be links to the underlying issues (in addition to the existing PR link)? 
Maybe something like:
```
Corrected an implementation error in `coreax.solvers.CaratheodoryRecombination`
which caused numerical instability when using either `CaratheodoryRecombination`
or `TreeRecombination` on GPU machines. 
(#852, #853, #874)
```